recipe/meta.yaml

```diff
@@ -1,28 +1,40 @@
-{% set name = "tokenizers" %}
 {% set version = "0.13.1" %}
 
 package:
-  name: {{ name|lower }}
+  name: tokenizers
```

Comment:
Can we keep these, please?

Reply:
I'm not a fan of these, because it's a "fake" variable - it can never be changed. It may help for the initial recipe generation, but serves no purpose afterward, and makes reading the recipe (and copying the source URL, e.g. for calculating a new hash) harder. If you insist, I'll change it, just very reluctantly 🙃